internal/pkg/externalplugins/config.go

```diff
@@ -422,3 +468,50 @@ func validateBlunderbuss(blunderbusses []TiCommunityBlunderbuss) error {
 
 	return nil
 }
+
+// validateLabelBlocker will return an error if the regex cannot compile or actions is illegal.
+func validateLabelBlocker(labelBlockers []TiCommunityLabelBlocker) error {
+	for _, labelBlocker := range labelBlockers {
+		for _, blockLabel := range labelBlocker.BlockLabels {
+			_, err := regexp.Compile(blockLabel.Regex)
+			if err != nil {
+				return err
+			}
+
+			err = validateLabelBlockerAction(blockLabel.Actions)
+			if err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
+// validateLabelBlockerAction used to check whether all actions filled in are allowed values.
+func validateLabelBlockerAction(actions []string) error {
+	allowActions := []string{
```

Comment:
Better make it be a const in the label blocker plugins.

Reply:
It will cause the import cycle.
![image](https://user-images.githubusercontent.com/5086433/106080072-5c141f80-6151-11eb-92d6-bc5e36418463.png)

Reply:
Maybe make a const in config.go?

Reply:
OK.